README.md

```diff
@@ -224,6 +224,21 @@ assert("a" in #{ a: 1, b: 2 });
 
 See the [appendix](./NS-Proto-Appendix.md) to learn more about the `Record` & `Tuple` namespaces.
 
+## `Record` and `Tuple` Wrapper Objects
+
+- `Object(record)` creates an instance of `Record`, which is the wrapper object for `record` values.
+- `Object(tuple)` creates an instance of `Tuple`, which is the wrapper object for `tuple` values.
+
+The `Record` prototype object is an ordinary object, is not a `Record` instance, and its prototype is `Object`.
+
+`record` values are not an instances of the `Record` prototype. A `Record` instance can be created via `Object(record)`.
+
+The `Tuple` prototype object is an ordinary object, is not a `Tuple` instance, and its prototype is `Object`.
+
+`tuple` values are not an instances of the `Tuple` prototype. A `Tuple` instance can be created via `Object(tuple)`.
+
+Accessing a member expression of a tuple or record via `.` or `[]` follows the standard [`GetValue`](https://www.ecma-international.org/ecma-262/#sec-getvalue) semantics, and implicitly converts to an instance of the corresponding wrapper type.
```

Comment:
Please link to tc39.es/ecma262/#sec-getvalue. Ecma publishes outdated snapshots.

Reply:
This section still should describe how those wrapped objects work, and why is it that they have the right properties. That's the interesting part!

Reply:
@littledan I updated to make this part more interesting. I based the behavior of `Tuple` on that of `String`. Looking forward to seeing if I'm on the right track! Thanks for your time.